README-SQL.md

```diff
@@ -35,15 +35,20 @@ appropriate values before the query is run.
 
 * `$$DAG$$` &ndash; the REDCap unique DAG ID of the user viewing the report
   * if the user is not in a DAG, *NULL* is used
+* `$$DATATABLE$$` &ndash; the redcap_data table used by the current project
+  * use `$$DATATABLE:pid$$`, where `pid` is a project ID, to get the redcap_data table used by

```

Comment:
Not quite sure which one is meant to use, $$DATATABLE$$ or $$DATATABLE:pid$$ or $$DATATABLE:42$$ ?